src/locale/nl.yml

```diff
@@ -773,7 +773,7 @@ feat:
       createLocation: Maak nieuwe locatie
       description: Omschrijving
       infoText: Je kunt klikken en slepen om de kaart te verschuiven en scrollen of je
-        vingers naar elkaar toe bewegen om te vergroten. Om een ​​nieuwe locatie
+        vingers naar elkaar toe bewegen om te vergroten. Om een nieuwe locatie
```

Comment:
Related to my other comment. We prefer not to make changes on these auto-generated files. 

```suggestion
```